packages/lisk-cryptography/src/nacl/slow.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import tweetnacl from 'tweetnacl';
+
+// eslint-disable-next-line prefer-destructuring
+export const box = tweetnacl.box;
```

Comment:
Oh sorry, didn't realise this was going to require us to disable a linting rule. I would say conforming to lint rules trumps this kind of uniformity.